### ISSUE TITLE
Calculate checksum fails if another segment is added with a gap between

### ISSUE THREAD

**Issue by Grazfather** [zznop/ghidra_scripts#1]
Will get a stack trace later, but basically if you add a segment, as I did for RAM, the checksum fails because it tries to read the data between the segments.

**Comment by zznop** [comment]
Yeah, the checksum plugin is written to calculate the checksum over the entire view. I need to modify the loader script to create a "ROM" segment and only calculate the CRC over the ROM
